docs/conversion_examples_gallery/recording/spikeglx.rst

```diff
@@ -7,7 +7,41 @@ Install NeuroConv with the additional dependencies necessary for reading SpikeGL
 
     pip install neuroconv[spikeglx]
 
-Convert ap or lf SpikeGLX streams to NWB using
+
+
+SpikeGLXConverter
+~~~~~~~~~~~~~~~~~
+
+We can easily convert all data stored in the native SpikeGLX folder structure to NWB using
+:py:class:`~neuroconv.converters.SpikeGLXConverterPipe`.
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from neuroconv.converters import SpikeGLXConverterPipe
+    >>>
+    >>> folder_path = f"{ECEPHY_DATA_PATH}/spikeglx/Noise4Sam_g0"
+    >>> converter = SpikeGLXConverterPipe(folder_path=folder_path)
+    >>>
+    >>> # Extract what metadata we can from the source files
+    >>> metadata = converter.get_metadata()
+    >>> # For data provenance we add the time zone information to the conversion
+    >>> session_start_time = metadata["NWBFile"]["session_start_time"].replace(tzinfo=tz.gettz("US/Pacific"))
+    >>> metadata["NWBFile"].update(session_start_time=session_start_time)
+    >>>
+    >>> # Choose a path for saving the nwb file and run the conversion
+    >>> nwbfile_path = output_folder / "my_spikeglx_session.nwb"
+    >>> converter.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata)
+    root pynwb.file.NWBFile at ...
+
+
+
+Single-stream
+~~~~~~~~~~~~~
+
+Defining a 'stream' as a single band on a single NeuroPixels probe, we can convert either an AP or LF SpikeGLX stream to NWB using
```

Comment:
Is band the right word here? Not that I find a better one. Maybe this is very clear for people that work making this recordings but is not for me. Other possibilites: modalities, recoding mode.

Reply:
Band is the word I hear most for talking about AP vs LF, i.e., 'AP band'; I don't actually know though lol

Maybe something for @alejoe91 to weigh in on